### ISSUE TITLE
resolution error when running runtime

### ISSUE THREAD

**Issue by sbouchet** [eclipse-linuxtools/org.eclipse.linuxtools#87]
Hi,
test case is to run a runtime with all linuxtools bundles with java17. the are equinox resolution errors logged (see below)
this is likely same as https://bugs.eclipse.org/bugs/show_bug.cgi?id=564339 

```

!ENTRY org.eclipse.linuxtools.docker.integration.tests 4 0 2022-08-05 12:45:59.713
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.linuxtools.docker.integration.tests [279]
  Bundle was not resolved because of a uses constraint violation.
  org.apache.felix.resolver.reason.ReasonException: Uses constraint violation. Unable to resolve resource org.eclipse.linuxtools.docker.integration.tests [osgi.identity; osgi.identity="org.eclipse.linuxtools.docker.integration.tests"; type="osgi.bundle"; version:Version="2.2.0.qualifier"; singleton:="true"] because it is exposed to package 'org.hamcrest' from resources org.hamcrest [osgi.identity; osgi.identity="org.hamcrest"; type="osgi.bundle"; version:Version="2.2.0.v20210711-0821"] and org.hamcrest.core [osgi.identity; osgi.identity="org.hamcrest.core"; type="osgi.bundle"; version:Version="1.3.0.v20180420-1519"] via two dependency chains.

Chain 1:
  org.eclipse.linuxtools.docker.integration.tests [osgi.identity; osgi.identity="org.eclipse.linuxtools.docker.integration.tests"; type="osgi.bundle"; version:Version="2.2.0.qualifier"; singleton:="true"]
    import: (osgi.wiring.package=org.hamcrest)
     |
    export: osgi.wiring.package: org.hamcrest
  org.hamcrest [osgi.identity; osgi.identity="org.hamcrest"; type="osgi.bundle"; version:Version="2.2.0.v20210711-0821"]

Chain 2:
  org.eclipse.linuxtools.docker.integration.tests [osgi.identity; osgi.identity="org.eclipse.linuxtools.docker.integration.tests"; type="osgi.bundle"; version:Version="2.2.0.qualifier"; singleton:="true"]
    require: (osgi.wiring.bundle=org.junit)
     |
    provide: osgi.wiring.bundle: org.junit
  org.hamcrest.core [osgi.identity; osgi.identity="org.hamcrest.core"; type="osgi.bundle"; version:Version="1.3.0.v20180420-1519"]
	at org.eclipse.osgi.container.Module.start(Module.java:463)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1847)
	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1840)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1781)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1745)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1667)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)

```

**Comment by akurtakov** [comment]
@sbouchet This one is fixed now, right?

**Comment by sbouchet** [comment]
yes, closed by #91
